com.ibm.streamsx.metrics/build.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed Materials - Property of IBM
+# Copyright IBM Corp. 2011, 2013
+# US Government Users Restricted Rights - Use, duplication or
```

Comment:
This is actually the wrong copyright for open source projects, remove the US Government restrictions since the code is released under ALv2.